pkg/scheduler/objects/application.go

```diff
@@ -1638,6 +1639,7 @@ func (sa *Application) addAllocationInternal(info *Allocation) {
 					zap.Error(err))
 			}
 		}
+		sa.hasPlaceholderAlloc = true
```

Comment:
NIT: this could move inside the if checking for zero allocated placeholders  at line 1626

Reply:
Done